src/components/containers/SingleArticle.jsx

```diff
@@ -115,7 +114,10 @@ class SingleArticle extends Component {
             <div className="heading-info">
               <div className="article-info">
                 <div className="author-image-box">
-                  <p className="author"> written by {author.username}</p>
+                  <p className="author">
+                    {' '}
+                    written by {author.username ? author.username.replace(/\d+/, '') : ''}
+                  </p>
```

Comment:
I believe the logic here is to remove the first set of numbers it finds in the string. Is it not possible for the username to be strictly numbers?